CPT331.Data.Migration/CPT331.Data.Migration.csproj

```diff
@@ -70,6 +70,12 @@
     <None Include="App.config">
       <SubType>Designer</SubType>
     </None>
+    <Content Include="CmdHelpers\dbprod.bat">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="CmdHelpers\dbdev.bat">
```

Comment:
If you run the migration exe with --help, or with an argument that isn't supported, it will tell you.